### PR TITLE
Adds `Padding: all sides minimal` style for text area and webform blocks

### LINKED ISSUE #6905: Implement small padding style for text area and webform blocks
# Story

As a SiteNow editor, I can use a small padding option style for text area and webform blocks. 

Currently we have:
Screenshot 2023-09-20 at 8 15 41 AM

# Proposed solution

Add `All sides: small` style that uses the existing `.block-padding__all--md` class.

### PR DESCRIPTION
**Pull request by bspeare**
Resolves https://github.com/uiowa/uiowa/issues/6905

In this pr:
- Adds `Padding: all sides minimal` style
  - Would `Padding: all sides compact` be better?
- Changes the padding group from multiselect to single select. 

# How to test

```
ddev blt frontend && ddev blt ds --site=sandbox.uiowa.edu && ddev drush @sandbox.local uli /test-sidebar
```
- Code review
- Test the webform block and the text area block on this page and confirm that the `Padding: all sides minimal` style is available and sets the block padding to `2rem`. 
- Test other pages and webform blocks to see if they are or are not affected. 
- Test with gray background plus minimal padding setting.

### CODE REVIEW

**Comment by cory-skeers** [comment]
When syncing admissions, I'm getting an error with the update hook:

```
>  [notice] Update started: sitenow_update_9072
>  [error]  in_array(): Argument #2 ($haystack) must be of type array, null given 
>  [error]  Update failed: sitenow_update_9072 
>  [notice] Update started: sitenow_dispatch_update_9001
>  [notice] Update completed: sitenow_dispatch_update_9001
>  [notice] Message: Sections with blocks of inline_block:uiowa_text_area records found in 
> node__layout_builder__layout: 391
> 
>  [notice] Message: Sections with blocks of inline_block:uiowa_text_area records found in 
> node_revision__layout_builder__layout: 3458
> 
>  [notice] Message: Tempstore records for sections with blocks of inline_block:uiowa_text_area 
> records found: 3
> 
>  [notice] Message: Sections with blocks of webform_block records found in 
> node__layout_builder__layout: 17
> 
 [error]  Update aborted by: sitenow_update_9072 
 [error]  Finished performing updates. 
 [Acquia\Blt\Robo\Tasks\DrushTask]  Exit code 1  Time 8.244s
 [error]  Failed to execute database updates!
For troubleshooting guidance and support, see https://docs.acquia.com/blt/support/ 
The command failed. This often indicates a problem with your configuration. Review the command output above for more detailed errors, and consider re-running with verbose output for more information.
 [error]  Command `drupal:update --site='admissions.uiowa.edu'` exited with code 1.
For troubleshooting guidance and support, see https://docs.acquia.com/blt/support/ 
Failed to run blt ds --site=admissions.uiowa.edu: exit status 1
```

**Comment by bspeare** [comment]
@cory-skeers if you are still here could you try that sync again?

**Comment by cory-skeers** [comment]
> @cory-skeers if you are still here could you try that sync again?

That did it! Was able to sync error-free.

**Review by e-marie-w — APPROVED** [review]
Code changes make sense. All sides minimal padding works as expected on text area blocks and webform blocks, including those with gray backgrounds. Existing blocks are unaffected, except for blocks where all sides and all sides extra were set - those are now set to only all sides extra as expected due to new single select. I also ran a quick blt dsa on a dozen sites and had no errors.

**Comment by pyrello** [comment]
@bspeare asked me to take a look at this with an eye towards how the option is being named. I'm going to test this as soon as I'm able to.

File: config/default/layout_builder_styles.style.block_padding_all.yml
Comment (pyrello) [inline config/default/layout_builder_styles.style.block_padding_all.yml]: I think we may want to set this via code. There are examples of this in `layout_builder_custom.module`.

**Comment by pyrello** [comment]
Screenshot 2023-09-29 at 9 44 18 AM

I'm not sure if this behavior is intended, but you typically wouldn't want to have a `(default)` option when also showing `-None-`.

**Comment by pyrello** [comment]
Are we intending to have other options besides different thickness for all sides? If not, then it seems like we could remove the "All sides" text from each option and just focus on the different padding levels.

**Comment by bspeare** [comment]
Thanks for the suggestions @pyrello.  I have updated the selections to look like this:

Screenshot 2023-09-29 at 2 26 56 PM

**Comment by pyrello** [comment]
I'm also wondering if we should change the `- None -` text to `None`, since none is actually an option instead of the absence of an option?

**Comment by bspeare** [comment]
Is the `- None -` part of layout builder styles?  Do we want to add a "None" style that is the default here instead?

**Comment by pyrello** [comment]
> Is the `- None -` part of layout builder styles? Do we want to add a "None" style that is the default here instead?

`- None -` is part of Layout Builder Styles. We would be converting the label in code.

**Comment by bspeare** [comment]
Here's the adjusted `None` label:

Screenshot 2023-09-29 at 3 16 00 PM

**Comment by joewhitsitt** [comment]
I've reviewed the commits since Marie's approval and Sean's feedback. Are you looking for a re-test of this @bspeare?

**Comment by bspeare** [comment]
I think we can merge without test but if you are able to test once more, that would be great!

**Comment by joewhitsitt** [comment]
yeah i'll pull down quick and test

File: config/default/layout_builder_styles.style.block_padding_all.yml
Comment (joewhitsitt) [reply config/default/layout_builder_styles.style.block_padding_all.yml]: Does this still need to be moved to layout_builder_custom.module?

Comment (bspeare) [reply config/default/layout_builder_styles.style.block_padding_all.yml]: I think adjusting the label to just `Default` allows us to bypass that?

Comment (joewhitsitt) [reply config/default/layout_builder_styles.style.block_padding_all.yml]: yeah sorry. didn't connect the remaining conversation.

**Comment by joewhitsitt** [comment]
Commits reviewed. Tested  and it seems to check out. I noticed that the dropdown order is None (0), Default (3rem), Minimal (2rem), Extra (6rem). Should Minimal go before Default in the list?

**Comment by bspeare** [comment]
Thanks @joewhitsitt.  I moved `Minimal` before `Default` and I will merge this once the build passes.

**Pull request merged**
